doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -714,29 +714,199 @@ general for HDDs, the bluestore throttle values are expected to be higher when
 
 
-Specifying  Max OSD Capacity
-----------------------------
+Override Max IOPS Capacity of an OSD
+------------------------------------
 
-The steps in this section may be performed only if you want to override the
-max osd capacity automatically set during OSD initialization. The option
-``osd_mclock_max_capacity_iops_[hdd, ssd]`` for an OSD can be set by running the
-following command:
+The steps in this section may be performed if you want to override the max IOPS
+capacity of an OSD that was automatically set during its initialization.
+The option ``osd_mclock_max_capacity_iops_[hdd, ssd]`` for an OSD can be
+overridden by running the following command:
 
-  .. prompt:: bash #
+.. prompt:: bash #
 
-     ceph config set osd.N osd_mclock_max_capacity_iops_[hdd,ssd] <value>
+  ceph config set osd.N osd_mclock_max_capacity_iops_[hdd,ssd] <value>
 
 For example, the following command sets the max capacity for a specific OSD
 (say "osd.0") whose underlying device type is HDD to 350 IOPS:
 
-  .. prompt:: bash #
+.. prompt:: bash #
 
-    ceph config set osd.0 osd_mclock_max_capacity_iops_hdd 350
+  ceph config set osd.0 osd_mclock_max_capacity_iops_hdd 350
 
 Alternatively, you may specify the max capacity for OSDs within the Ceph
 configuration file under the respective [osd.N] section. See
 :ref:`ceph-conf-settings` for more details.
 
+Global Override of Max IOPS Capacity for multiple OSDs
+------------------------------------------------------
+
+Depending on the requirement, the max IOPS capacity of multiple OSDs may be
+overridden by a global config specification. This can be done in case multiple
+OSDs are created on the same type of underlying hardware. There are two
+scenarios to consider in order to effect this change and are discussed below.
+The examples use :confval:`osd_mclock_max_capacity_iops_hdd`.
+
+Scenario 1: Config global set before deploying OSDs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a global value is set before deploying the OSDs, the benchmark test during
+OSD deployment is skipped thus ensuring that the globally set value is in effect.
+
+The config global may be set before deploying OSDs but after enabling the Ceph
+monitor(s) using:
+
+.. prompt:: bash #
+
+  ceph config set global osd_mclock_max_capacity_iops_hdd 370
+
+Alternatively, the IOPS capacity may be specified under the *[global]* section of
+the Ceph configuration file during cluster deployment.
+
+A config dump after the OSD is deployed shows the global value in effect
+shown below:
+
+.. prompt:: bash #
+
+  ceph config dump
+
+::
+
+  WHO     MASK  LEVEL  OPTION                            VALUE       RO
+  global        basic  osd_mclock_max_capacity_iops_hdd  370.000000    
+
+.. prompt:: bash #
+
+  ceph config show osd.0 | grep osd_mclock
+
+::
+
+  osd_mclock_max_capacity_iops_hdd                 370.000000                               mon
+  ...
+
+Scenario 2: Config global set after deploying OSDs with individual values
```

Comment:
The user did not deploy osds "with individual values" -- it's the default behavior and the user will not be aware of it.

Reply:
Structuring this as two scenarios really isn't useful because the user will not have consciously chosen to set individual values.  Instead, the first step of the instructions should be simply be to check for and clear all existing individual osd config (there might be none).

Reply:
Is there a way to set different values for different classes of OSD?  e.g. ssd vs hdd?

Reply:
Restructured the content to a single section with focus on steps to check and remove existing individual values and then applying the global value. I have added a note to highlight that the steps are equally applicable for ssd device class that has a different config option.

Reply:
LGTM, we can merge with athanatos' approval.